Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.23-alpine3.20 AS builder
+
+ENV GO111MODULE=on
+
+WORKDIR /gemini
+
+COPY . .
+
+RUN apk add build-base musl-dev \
+    && go build -ldflags="-s -w" -o build/gemini ./cmd/gemini
+
+FROM alpine:3.20 AS production
```

Comment:
No need to base it of the `alpine` do it from the `busybox`

Reply:
Done